### PR TITLE
[#9] Update to Rails 5

### PR DESCRIPTION
**Pull request by syndbg**
* Fixes deprecation warnings
* Allows bundle install with rails 5 gem

Addresses #9 

@gregnavis @crmaxx

### CODE REVIEW

**Comment by syndbg** [comment]
ping @gregnavis :)

**Comment by gregnavis** [comment]
Thanks for the PR! I'm sorry for radio silence. I'll take a look next week. Stay tuned.

File: active_record_doctor.gemspec [hunk 1]
```diff
@@ -16,9 +16,9 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE.txt", "Rakefile", "README.md"]
   s.test_files = Dir["test/**/*"]
 
-  s.add_dependency "rails", "~> 4.2"
+  s.add_dependency "rails", "< 5.1", ">= 4.2"
 
-  s.add_development_dependency "rails", "~> 4.2"
+  s.add_development_dependency "rails", "< 5.0", ">= 4.2"
```
Comment (gregnavis) [inline active_record_doctor.gemspec]: Shouldn't it read `"< 5.1"`? Currently it reads `"< 5.0"`.

Comment (syndbg) [inline active_record_doctor.gemspec]: Surprisingly it installed without any problems with Rails 5.x, however

`< 5.1` sure is more future proof.

File: Gemfile.lock [hunk 2]
```diff
@@ -107,9 +109,9 @@ PLATFORMS
 
 DEPENDENCIES
   active_record_doctor!
-  rails (~> 4.2)
+  rails (>= 4.2, < 5.0)
```
Comment (gregnavis) [inline Gemfile.lock]: It seems we need to rerun `bundle install`. Could you confirm, @syndbg? Also see https://travis-ci.org/gregnavis/active_record_doctor/builds/161303757

**Comment by gregnavis** [comment]
Thanks a lot @syndbg!

**Pull request merged**